File: api-BabyCare/app/Models/Pinjaman.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Pinjaman extends Model
{
    use HasFactory;

    protected $fillable = [
        'id_peminjam',
        'id_barang',
        'tgl_pinjam',
        'tgl_kembali',
        'status',
        'denda',
    ];
}

File: BabyCareX/app/src/main/java/com/konyol/babycarex/data/model/Pinjaman.kt
package com.konyol.babycarex.data.model

data class Pinjaman(
    val id: Int? = null,
    val id_peminjam: Int,
    val id_barang: Int,
    val tgl_pinjam: String,
    val tgl_kembali: String,
    val status: String,
    val denda: Int,
    val created_at: String? = null,
    val updated_at: String? = null
)
